test/bulk_insert/worker_test.rb

```diff
@@ -121,6 +121,58 @@ class BulkInsertWorkerTest < ActiveSupport::TestCase
     assert_equal true, hello.happy?
   end
 
+  test "save! adds to results when there are existing results" do
+    worker = BulkInsert::Worker.new(
+      Testing.connection,
+      Testing.table_name,
+      %w(greeting age happy created_at updated_at color)
+    )
+    worker.add greeting: "first"
+    worker.add geeting: "second"
```

Comment:
Does `geeting` column exist? I'm not sure why a result is being returned for this.

Reply:
Good catch. For that one, it's probably inserting a record with nulls in an columns.

Reply:
Which is why a result is still returned

Reply:
Ya just confirmed that's what's going on. I'll fix the typo though.